### PR TITLE
Fix issues in `network-interaction.md`

### PR DESCRIPTION
**Pull request by rdulmina**
## Purpose
$title

### CODE REVIEW

**Review by lochana-chathura — APPROVED** [review]

**Comment by lochana-chathura** [comment]
@rdulmina try formatting [Consuming Services - Client Objects](http://dev.ballerina.io/learn/distinctive-language-features/network-interaction/#consuming-services---client-objects) and [Listener Declaration](http://dev.ballerina.io/learn/distinctive-language-features/network-interaction/#listener-declaration) code samples. There is a small diff.

File: learn/guides/distinctive-language-features/network-interaction.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The first is a service object. These objects are defined by applications and con
 
 The second is a Listener object. It receives network input and makes calls to remote methods of attached service objects. Service objects are attached to Listeners, which are registered with a module that is the third key entity in providing network services.
 
-Modules have a lifecycle and are initialized on program startup. Post initializations, the registered listeners start up, and when the program shuts down, the registered listeners also shut down.
+Modules have a lifecycle and are initialized on program startup. Post initializations, the registered listeners startup, and when the program shuts down, the registered listeners also shut down.
```
Comment (MaryamZi) [inline learn/guides/distinctive-language-features/network-interaction.md:63]: The previous form (the verb) was correct

```suggestion
Modules have a lifecycle and are initialized on program startup. Post initialization, the registered listeners start up, and when the program shuts down, the registered listeners also shut down.
```

Or we should say
```suggestion
Modules have a lifecycle and are initialized on program startup. Post initialization, the registered listeners are started up, and when the program shuts down, the registered listeners are also shut down.
```

File: learn/guides/distinctive-language-features/network-interaction.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ This also registers the Listener with the enclosing module. If the ``new`` expre
 
 As mentioned earlier, all modules have a lifecycle, so let's understand a few aspects of the lifecycle.
 
-During initialization, all module listeners are registered. The initialization sequence is ordered such that if module A imports module B, then module A is initialized after module B. The initialization phase ends by calling the ``main`` function if present.
+During initialization, all module listeners are registered. The initialization sequence is ordered such that if module *A* imports module *B*, then module *A* is initialized after module *B*. The initialization phase ends by calling the ``main`` function if present.
```
Comment (MaryamZi) [inline learn/guides/distinctive-language-features/network-interaction.md:81]:
```suggestion
During initialization, all module listeners are registered. The initialization sequence is ordered such that if module ``A`` imports module ``B``, then module ``A`` is initialized after module ``B``. The initialization phase ends by calling the ``main`` function if present.
```

File: learn/guides/distinctive-language-features/network-interaction.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ var obj = object {
 string greeting = obj.greet();
 ```
 
-In the above code example, the ``object`` keyword is used in the expression that initializes the **``obj``** variable. This is called an object constructor. It has one method **``greet()``** that returns a string "Hello World". So there is no class involved in this case.
+In the above code example, the ``object`` keyword is used in the expression that initializes the **``obj``** variable. This is called an object constructor. It has one method **``greet()``** that returns a string *"Hello World"*. So there is no class involved in this case.
```
Comment (MaryamZi) [inline learn/guides/distinctive-language-features/network-interaction.md:119]:
```suggestion
In the above code example, the ``object`` keyword is used in the expression that initializes the **``obj``** variable. This is called an object constructor. It has one method **``greet()``** that returns a string ``"Hello World"``. So there is no class involved in this case.
```

**Review by MaryamZi — APPROVED** [review]

**Review by lochana-chathura — APPROVED** [review]

**Pull request merged**